<launch>

  <node pkg="giskardpy_ros" type="collision_matrix_tool.py" name="collision_matrix_tool" output="screen"/>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find giskardpy_ros)/launch/rviz_config/collision_matrix_tool.rviz" />

</launch>
